templates/compose/ente.yaml

```
@@ -0,0 +1,189 @@
+      ENTE_HTTP_USE_TLS: ${ENTE_HTTP_USE_TLS:-false}
+
+      ENTE_APPS_PUBLIC_ALBUMS: ${SERVICE_URL_WEB_3002:-http://localhost:3002}
+      ENTE_APPS_CAST: ${SERVICE_URL_WEB_3004:-http://localhost:3004}
+      ENTE_APPS_ACCOUNTS: ${SERVICE_URL_WEB_3001:-http://localhost:3001}
+      ENTE_APPS_PUBLIC_LOCKER: ${SERVICE_URL_WEB_3003:-http://localhost:3003}
+      ENTE_APPS_CUSTOM_DOMAIN_CNAME: ${ENTE_APPS_CUSTOM_DOMAIN_CNAME}
+
+      ENTE_DB_HOST: ${ENTE_DB_HOST:-postgres}
+      ENTE_DB_PORT: ${ENTE_DB_PORT:-5432}
+      ENTE_DB_NAME: ${ENTE_DB_NAME:-ente_db}
+      ENTE_DB_SSLMODE: ${ENTE_DB_SSLMODE:-disable}
+      ENTE_DB_USER: ${SERVICE_USER_POSTGRES:-pguser}
+      ENTE_DB_PASSWORD: ${SERVICE_PASSWORD_POSTGRES}
+
+      ENTE_KEY_ENCRYPTION: ${MUSEUM_ENCRYPTION_KEY}
+      ENTE_KEY_HASH: ${MUSEUM_HASH_KEY}
+
+      ENTE_JWT_SECRET: ${MUSEUM_JWT_KEY}
+
+      ENTE_INTERNAL_SILENT: ${ENTE_INTERNAL_SILENT:-false}
+      ENTE_INTERNAL_HEALTH_CHECK_URL: ${ENTE_INTERNAL_HEALTH_CHECK_URL}
+      ENTE_INTERNAL_HARDCODED_OTT_EMAILS: ${ENTE_INTERNAL_HARDCODED_OTT_EMAIL}
+      ENTE_INTERNAL_HARDCODED_OTT_LOCAL_DOMAIN_SUFFIX: ${ENTE_INTERNAL_HARDCODED_OTT_LOCAL_DOMAIN_SUFFIX}
+      ENTE_INTERNAL_HARDCODED_OTT_LOCAL_DOMAIN_VALUE: ${ENTE_INTERNAL_HARDCODED_OTT_LOCAL_DOMAIN_VALUE}
+      ENTE_INTERNAL_ADMINS: ${ENTE_INTERNAL_ADMINS}
+      ENTE_INTERNAL_ADMIN: 1580559962386438
+      ENTE_INTERNAL_DISABLE_REGISTRATION: ${ENTE_INTERNAL_DISABLE_REGISTRATION:-false}
+
+      ENTE_S3_ARE_LOCAL_BUCKETS: ${ENTE_S3_ARE_LOCAL_BUCKETS:-true}
+      ENTE_S3_USE_PATH_STYLE_URLS: ${ENTE_S3_USE_PATH_STYLE_URLS:-true}
+
+      ENTE_S3_B2_EU_CEN_KEY: ${SERVICE_USER_MINIO}
+      ENTE_S3_B2_EU_CEN_SECRET: ${SERVICE_PASSWORD_MINIO}
+      ENTE_S3_B2_EU_CEN_ENDPOINT: ${SERVICE_URL_MINIO}:3200
+      ENTE_S3_B2_EU_CEN_REGION: ${PRIMARY_STORAGE_REGION:-eu-central-2}
+      ENTE_S3_B2_EU_CEN_BUCKET: ${PRIMARY_STORAGE_BUCKET:-b2-eu-cen}
+      ENTE_S3_B2_EU_CEN_ARE_LOCAL_BUCKETS: ${PRIMARY_STORAGE_ARE_LOCAL_BUCKETS:-false}
+      ENTE_S3_B2_EU_CEN_USE_PATH_STYLE_URLS: ${PRIMARY_STORAGE_USE_PATH_STYLE_URLS:-false}
+
+      ENTE_S3_WASABI_EU_CENTRAL_2_V3_KEY: ${SERVICE_USER_MINIO}
+      ENTE_S3_WASABI_EU_CENTRAL_2_V3_SECRET: ${SERVICE_PASSWORD_MINIO}
+      ENTE_S3_WASABI_EU_CENTRAL_2_V3_ENDPOINT: ${SERVICE_URL_MINIO}:3200
+      ENTE_S3_WASABI_EU_CENTRAL_2_V3_REGION: ${SECONDARY_STORAGE_REGION:-eu-central-2}
+      ENTE_S3_WASABI_EU_CENTRAL_2_V3_BUCKET: ${SECONDARY_STORAGE_BUCKET:-wasabi-eu-central-2-v3}
+      ENTE_S3_WASABI_EU_CENTRAL_2_V3_ARE_LOCAL_BUCKETS: ${SECONDARY_STORAGE_ARE_LOCAL_BUCKETS:-false}
+      ENTE_S3_WASABI_EU_CENTRAL_2_V3_USE_PATH_STYLE_URLS: ${SECONDARY_STORAGE_USE_PATH_STYLE_URLS:-false}
+      ENTE_S3_WASABI_EU_CENTRAL_2_V3_COMPLIANCE: ${SECONDARY_STORAGE_COMPLIANCE:-true}
+
+      ENTE_S3_SCW_EU_FR_V3_KEY: ${SERVICE_USER_MINIO}
+      ENTE_S3_SCW_EU_FR_V3_SECRET: ${SERVICE_PASSWORD_MINIO}
+      ENTE_S3_SCW_EU_FR_V3_ENDPOINT: ${SERVICE_URL_MINIO}:3200
```

```suggestion
      ENTE_S3_SCW_EU_FR_V3_ENDPOINT: ${SERVICE_URL_MINIO}:3200
```
If I am not wrong, uploads will be to a signed url in Minio, so this config will not work with the port when you are behind https as the port is hard coded.